evap/static/scss/components/_card.scss

```diff
@@ -16,6 +16,14 @@
     background-color: rgba($evap-yellow, 0.5);
 }
 
+.card-outline-danger {
```

Comment:
maybe you could update the other definitions as well so we have the same structure for all cards